wild/Cargo.toml

```diff
@@ -5,6 +5,9 @@ edition = "2021"
 
 [dependencies]
 wild_lib = { version = "0.2.0", path = "../wild_lib" }
+anyhow = "1.0.93"
+libc = "0.2"
+tempdir = "0.3.7"
```

Comment:
Just an experiment - does wild already have some standard method for creating tempfiles or tempdirs that could be reused?